---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Integrative genomics of the mammalian alveolar macrophage response to intracellular
  mycobacteria
subtitle: ''
summary: ''
authors:
- Thomas J. Hall
- Michael P. Mullen
- GillianP.McHugo
- Kate E. Killick
- Siobhán C. Ring
- Donagh P. Berry
- Carolina N. Correia
- John A. Browne
- Stephen V. Gordon
- David E. MacHugh
tags: []
categories: []
date: '2021-05-13'
lastmod: ''
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: ''
publication_types:
- '7'
abstract: '**Background**<br>Bovine TB (bTB), caused by infection with *Mycobacterium bovis*,
  is a major endemic disease affecting global cattle production. The key innate immune
  cell that first encounters the pathogen is the alveolar macrophage, previously shown
  to be substantially reprogrammed during intracellular infection by the pathogen.
  Here we use differential expression, and correlation- and interaction-based network
  approaches to analyse the host response to infection with *M. bovis* at the transcriptome
  level to identify core infection response pathways and gene modules. These outputs
  were then integrated with genome-wide association study (GWAS) data sets to enhance
  detection of genomic variants for susceptibility/resistance to *M. bovis* infection.
  <br><br>**Results**<br>The host gene expression data consisted of RNA-seq data from bovine alveolar
  macrophages (bAM) infected with *M. bovis* at 24 and 48 h post-infection (hpi) compared
  to non-infected control bAM. These RNA-seq data were analysed using three distinct
  computational pipelines to produce six separate gene sets: 1) DE genes filtered
  using stringent fold-change and *P*-value thresholds (DEG-24: 378 genes, DEG-48: 390
  genes); 2) genes obtained from expression correlation networks (CON-24: 460 genes,
  CON-48: 416 genes); and 3) genes obtained from differential expression networks
  (DEN-24: 339 genes, DEN-48: 495 genes). These six gene sets were integrated with
  three bTB breed GWAS data sets by employing a new genomics data integration tool—*gwinteR*.
  Using GWAS summary statistics, this methodology enabled detection of 36, 102 and
  921 prioritised SNPs for Charolais, Limousin and Holstein-Friesian, respectively.
  <br><br>**Conclusions**<br>The results from the three parallel analyses showed that the three
  computational approaches could identify genes significantly enriched for SNPs associated
  with susceptibility/resistance to *M. bovis* infection. Results indicate distinct
  and significant overlap in SNP discovery, demonstrating that network-based integration
  of biologically relevant transcriptomics data can leverage substantial additional
  information from GWAS data sets. These analyses also demonstrated significant differences
  among breeds, with the Holstein-Friesian breed GWAS proving most useful for prioritising
  SNPS through data integration. Because the functional genomics data were generated
  using bAM from this population, this suggests that the genomic architecture of bTB
  resilience traits may be more breed-specific than previously assumed.'
publication: 'Collection. Figshare'
doi: 10.6084/m9.figshare.c.5422454
links:
- name: Link
  url: https://springernature.figshare.com/collections/Integrative_genomics_of_the_mammalian_alveolar_macrophage_response_to_intracellular_mycobacteria/5422454
  icon_pack: fas
  icon: external-link-square-alt
- name: Download
  url: https://springernature.figshare.com/collections/5422454/1/citations/text/figshare-datacite
  icon_pack: fas
  icon: file-alt
---
{{< metrics >}}