---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Integrative genomics of the mammalian alveolar macrophage response to intracellular
  mycobacteria
subtitle: ''
summary: ''
authors:
- Thomas J. Hall
- Michael P. Mullen
- GillianP.McHugo
- Kate E. Killick
- Siobh√°n C. Ring
- Donagh P. Berry
- Carolina N. Correia
- John A. Browne
- Stephen V. Gordon
- David E. MacHugh
tags: []
categories: []
date: '2020-12-10'
lastmod: ''
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: ''
publication_types:
- '5'
abstract: '**Background**<br>Bovine TB (BTB), caused by infection with *Mycobacterium bovis*,
  is a major endemic disease affecting global cattle production, particularly in many
  developing countries. The key innate immune that first encounters the pathogen is
  the alveolar macrophage, previously shown to be substantially reprogrammed during
  intracellular infection by the pathogen. Here we use differential expression, and
  correlation- and interaction-based network approaches to analyse the host response
  to infection with *M. bovis* at the transcriptome level to identify core infection
  response pathways and gene modules. These outputs were then integrated with genome-wide
  association study (GWAS) data sets to enhance detection of genomic variants for
  susceptibility/resistance to *M. bovis* infection.<br><br>**Results**<br>The host gene expression
  data consisted of bovine RNA-seq data from alveolar macrophages infected with *M.
  bovis* at 24 and 48 hours post-infection. These RNA-seq data were analysed using
  three distinct analysis pipelines and novel response pathways and modules were further
  refined using cross-comparison and integration of the results. First, a differential
  expression analysis was carried out to determine the most significantly differentially
  expressed (DE) genes between conditions at each time point. Second, two networks
  were constructed at each time point using gene correlation patterns to determine
  changes in expression across conditions. Functional sub-modules within each correlation
  network were selected by statistical criteria for modularity. Third, a base gene
  interaction network of the mammalian host response to mycobacterial infection was
  generated using the GeneCards database and InnateDB. Differential gene expression
  data were superimposed on this base network to extract functional modules of interconnected
  DE genes.<br><br>**Conclusions**<br>Bovine GWAS data was obtained from a published BTB susceptibility/resistance
  study. The results from the three parallel analyses were integrated with this data
  to determine which of the three approaches identified genes significantly enriched
  for SNPs associated with susceptibility/resistance to *M. bovis* infection. Results
  indicate distinct and significant overlap in SNP discovery, demonstrating that network-based
  integration of biologically relevant transcriptomics data can leverage substantial
  additional information from GWAS data sets.'
publication: 'Research Square'
doi: 10.21203/rs.3.rs-121955/v1
links:
- name: Link
  url: https://www.researchsquare.com/article/rs-121955/v1
  icon_pack: fas
  icon: external-link-square-alt
- name: Download
  url: https://www.researchsquare.com/article/rs-121955/v1.pdf?c=1631880774000
  icon_pack: fas
  icon: file-pdf
---
{{< metrics >}}