---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Integrative genomics of the host response to ğ˜”ğ˜ºğ˜¤ğ˜°ğ˜£ğ˜¢ğ˜¤ğ˜µğ˜¦ğ˜³ğ˜ªğ˜¶ğ˜® ğ˜£ğ˜°ğ˜·ğ˜ªğ˜´ in bovine
  alveolar macrophages
subtitle: ''
summary: ''
authors:
- Thomas J. Hall
- Michael P. Mullen
- Kate E. Killick
- GillianP.McHugo
- John A. Browne
- Carolina N. Correia
- Stephen V. Gordon
- David E. MacHugh
tags: []
categories: []
date: '2019-07-07'
lastmod: ''
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: ''
publication_types:
- '3'
abstract: Bovine TB (BTB), caused by infection with *Mycobacterium bovis*, is a major
  endemic disease affecting global cattle production. The key innate immune cell that
  first encounters the pathogen is the alveolar macrophage, which we have previously
  shown to be substantially reprogrammed during intracellular infection by M. bovis.
  In the current study we used differential expression with correlation- and interaction-based
  network approaches to analyze the macrophage transcriptional response to infection
  with *M. bovis* to identify core infection response pathways and gene modules. These
  outputs were then integrated with genome-wide association study (GWAS) data sets
  to enhance detection of genomic variants for susceptibility/resistance to *M. bovis*
  infection. The host gene expression data consisted of bovine RNA-seq data from alveolar
  macrophages infected with M. bovis at 24 and 48 h post-infection. These RNA-seq
  data were analyzed using 3 distinct analysis pipelines; novel response pathways
  and modules were further refined using cross-comparison and integration of the results.
  First, a differential expression analysis was carried out to determine the most
  significantly differentially expressed (DE) genes between conditions at each time
  point. Second, 2 networks were constructed at each time point using gene correlation
  patterns to determine changes in expression across conditions. Functional sub-modules
  within each correlation network were selected by statistical criteria for modularity.
  Third, a gene interaction base network of the mammalian host response to mycobacterial
  infection was generated using the GeneCards database (www.genecards.com) and InnateDB
  (www.innatedb.com). Differential gene expression data were superimposed on this
  base network to extract functional modules of interconnected DE genes. Bovine GWAS
  data were obtained from a published BTB susceptibility/resistance study. The results
  from the 3 parallel analyses were integrated with this data to determine which of
  the 3 approaches identified genes significantly enriched for SNPs associated with
  susceptibility/resistance to *M. bovis* infection. Our results indicate distinct and
  significant overlap in SNP discovery and demonstrate that network-based integration
  of relevant transcriptomics data can leverage substantial additional information
  from GWAS data sets.
publication: '37th International Society for Animal Genetics Conference. ResearchGate'
doi: 10.13140/rg.2.2.17194.36807
links:
- name: Link
  url: https://www.researchgate.net/publication/335061982_Integrative_genomics_of_the_host_response_to_Mycobacterium_bovis_in_bovine_alveolar_macrophages
  icon_pack: fas
  icon: external-link-square-alt
- name: Download
  url: https://www.researchgate.net/profile/Thomas-Hall-4/publication/335061982_Integrative_genomics_of_the_host_response_to_Mycobacterium_bovis_in_bovine_alveolar_macrophages/links/5d4d399f92851cd046ad8228/Integrative-genomics-of-the-host-response-to-Mycobacterium-bovis-in-bovine-alveolar-macrophages
  icon_pack: fas
  icon: file-powerpoint
---
{{< metrics >}}